Error: Navigation lists must be nested in a `nav` element
Error: Current page must be indicated using `aria-current="page"`
Error: Style changes (such as bold) must not be the only means of indicating a link is the current page
Error: Inline styles should be avoided in favor of external CSS, in this case using the `[aria-current=page]` selector
diff --git a/output/Basic Navigation with Current Page/2/1d43372b-f16b-43c7-83e7-464cf448b4b5.html b/output/Basic Navigation with Current Page/2/1d43372b-f16b-43c7-83e7-464cf448b4b5.html
index ccb9774..ab592f0 100644
--- a/output/Basic Navigation with Current Page/2/1d43372b-f16b-43c7-83e7-464cf448b4b5.html	
+++ b/output/Basic Navigation with Current Page/2/1d43372b-f16b-43c7-83e7-464cf448b4b5.html	
@@ -1,11 +1,13 @@
-<ul>
-  <li><a href="/">Home</a></li>
-  <li>
-    <a href="/products" style="font-weight: bold;">Products</a>
-    <ul>
-      <li><a href="/products/frobs">Frobs</a></li>
-      <li><a href="/products/thrips">Thrips</a></li>
-    </ul>
-  </li>
-  <li><a href="/services">Services</a></li>
-</ul>
\ No newline at end of file
+<nav>
+  <ul>
+    <li><a href="/">Home</a></li>
+    <li>
+      <a href="/products" aria-current="page">Products</a>
+      <ul>
+        <li><a href="/products/frobs">Frobs</a></li>
+        <li><a href="/products/thrips">Thrips</a></li>
+      </ul>
+    </li>
+    <li><a href="/services">Services</a></li>
+  </ul>
+</nav>
\ No newline at end of file
